.github/scripts/label-downstream-cluster.sh

```diff
@@ -2,7 +2,7 @@
 
 set -euxo pipefail
 
-ns=${FLEET_E2E_NS-fleet-default}
+ns=${FLEET_E2E_NS_DOWNSTREAM-fleet-default}
```

Comment:
I have had issues starting the test environment from the dev scripts _and_ running them with the same values of the those two environment variables:

* FLEET_E2E_NS
* FLEET_E2E_NS_DOWNSTREAM

With this change I've been able to start and run the tests with the same values of those env vars.